src/sidebar/services/tags.js

```diff
@@ -41,7 +41,7 @@ export class TagsService {
     //   (e.g. tag "pink banana" matches query "ban"), OR
     // * tag has substring query occurring after a non-word character
     //   (e.g. tag "pink!banana" matches query "ban")
```

Comment:
Yep, I think this makes sense, and I'll go ahead and convert it over and add more tests. 

But `words.any` should be `words.some` correct?

Reply:
> But words.any should be words.some correct?

Gah, yes. 🤦‍♂️

Reply:
I actually think "any" makes more sense, I think we should tell the js gods they are wrong :)

Reply:
I don't know the history of this specific method, but sometimes the standards committee are constrained by the need to avoid breaking existing websites. For example `array.includes` was originally going to be called `contains` but that turned out [to break existing code](https://github.com/tc39/proposal-Array.prototype.includes).